iphone/Classes/TiUIListView.m

```diff
@@ -92,6 +92,7 @@ - (void)dealloc
     [_headerViewProxy setProxyObserver:nil];
     [_footerViewProxy setProxyObserver:nil];
     [_pullViewProxy setProxyObserver:nil];
+    searchController.delegate = nil;
```

Comment:
Use space notation for setters rather than dots.
`[searchController setDelegate:nil];`